Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/deltaonboarding/ui/DeltaOnboardingFragmentViewModel.kt

```diff
@@ -18,14 +19,17 @@ class DeltaOnboardingFragmentViewModel @AssistedInject constructor(
     private val settings: CWASettings,
     private val analyticsSettings: AnalyticsSettings,
     private val traceLocationSettings: TraceLocationSettings,
-    private val contactDiarySettings: ContactDiarySettings,
+    private val contactDiaryUiSettings: ContactDiaryUiSettings,
```

Comment:
Can use use the term 'contact journal' everywhere (contact diary is not proper English)

Reply:
No, I wouldn't do the contactDiary -> contactJournal renaming in this PR, this should be done in its own ticket.
contactDiary naming spreads around the whole codebase and just changing it here would leave ambiguous naming.